motor_selective_stop_signal/experiment.js

```diff
@@ -63,6 +63,36 @@ function assessPerformance() {
 
 }
 
+var getSelectiveFeedback = function(){
+	var global_trial = jsPsych.progress().current_trial_global
+	if(jsPsych.data.getDataByTrialIndex(global_trial - 5).exp_stage == 'practice'){
+		var data_length = 60
```

Comment:
To fix these linting errors just define data_length at the top of the function "var data_length = 0") and then set it with the conditional "data_length = 60"